common/app/views/fragments/labourLiverpoolSeries.scala.html

```diff
@@ -9,16 +9,16 @@
 @defining((
```

Comment:
Should we rename this file to `exploreSeries.scala.html`? Or is this template still going to be specific to Labour & Liverpool?